dali/operators/image/color/brightness_contrast.cc

```diff
@@ -24,42 +24,71 @@ using TheKernel = kernels::MultiplyAddCpu<Out, In, 3>;
 }  // namespace
 
 
-DALI_SCHEMA(BrightnessContrast)
-    .DocStr(R"code(Adjusts the brightness and contrast of the images based on the following
-formula::
-
-  out = brightness_shift * output_range + brightness * (grey + contrast * (in - grey))
-
-Where:
-
--	The output_range is 1 for float outputs or the maximum positive value for integral types.
--	Grey denotes the value of 0.5 for ``float``, 128 for ``uint8``, and 16384 for ``int16``, and so on.
+DALI_SCHEMA(Brightness)
+    .DocStr(R"code(Adjusts the brightness of the images based on the following formula::
```

Comment:
This won't work well with the operators table, the first sentence should be short to fit into the table:
```suggestion
    .DocStr(R"code(Adjusts the brightness of images.
    
The brightness is adjusted based on the following formula::
```

Reply:
done